content/wissen/test4/index.md

```diff
@@ -0,0 +1,32 @@
+---
+draft: true
+
+title: "Test page 4"
+description: "How to set up a new branch, make changes and make a pullrequest"
+topics: ["test", "abc"]
+authors:
+  - "fackerl"
+series: ["Strommärkte"]
+series_order: 2
+weight: 2
+---
+
+# Test
+This is a test
+
+## Creating new branch
+Test
```

Comment:
Please make sure to have an empty line after each header line.

See the following `markdownlint` warning that VSCode should show you there (highlighting in yellow that there's a warning, and explaining what's wrong):

![image](https://github.com/user-attachments/assets/a82b29a7-bf96-4930-89e4-8eba1deaaedc)

If you don't see this, go to "extensions" in the left-side-menu-bar, search for `markdownlint` and install it. Should automatically check all your markdown files while you type.

You can then also open the "PROBLEMS" tab in the "terminal" section, which contains all these warnings in one place and you can "click and jump" to them:

![image](https://github.com/user-attachments/assets/4ef285aa-7491-41c1-8ab3-a0080b5e5d8c)

Reply:
Thanks, I installed it and now it shows me the warnings